src/model/LabyrinthMatrix.java

```diff
@@ -1,73 +0,0 @@
-package model;
-import java.io.File;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Scanner;
-
-public class LabyrinthMatrix extends LabyrinthImpl {
-
-	private int matrix[][];
-	private int  freeValue, wallValue;
-
-	public LabyrinthMatrix(int rows, int columns) {
-		this.rows = rows;
-		this.columns = columns;
-
-		matrix = new int[rows][columns];
-	}
-
-	public LabyrinthMatrix() {
-	}
-
-	public void loadFromFile(String path) {
-		Scanner sc;
-		try {
-			sc = new Scanner(new File(path));
-			rows = sc.nextInt();
-			columns = sc.nextInt();
-			matrix = new int[rows][columns];
-			for (int i = 0; i < rows; i++)
-				for (int j = 0; j < columns; j++) {
-					matrix[i][j] = sc.nextInt();
-					if (matrix[i][j] == startValue)
-						start = new Position(i, j);
-					if (matrix[i][j] == finishValue)
-						finish = new Position(i, j);
-				}
-			sc.close();
```

Comment:
I actually tried doing that, but if an exception is thrown, you have no scanner to close. If the scanner is initialized inside the try block and closed inside the finally block, I'm getting a "variable may not have been initialized" error.